src/main/java/com/example/CarDealership/CarDealershipApplication.java

```diff
@@ -0,0 +1,42 @@
+package com.example.CarDealership;
+
+import com.example.CarDealership.config.DataSourceProperties;
+import com.example.CarDealership.models.Dealership;
+import com.example.CarDealership.models.Vehicle;
+import com.example.CarDealership.repositories.DealershipRepository;
+import com.example.CarDealership.repositories.VehicleRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+import javax.xml.crypto.Data;
+import java.util.List;
+
+@SpringBootApplication
+public class CarDealershipApplication implements CommandLineRunner {
+
+	@Autowired
+	private DealershipRepository dealershipRepository;
+
+	@Autowired
```

Comment:
Why is data source properties needed here?

Reply:
hahaha it is not, thank you